### ISSUE TITLE
[Feature Request] Option to disable background operations

### ISSUE THREAD

**Issue by fref** [ForkIssues/TrackerWin#553]
Sometimes, my computer becomes unresponsive because Fork invoked some large git operation in the background, on its own (git status porcelain, stuff like that is one I could catch). I'd rather have it refresh on demand (F5) than having it doing it on its own.

**Comment by DanPristupov** [comment]
File -> Preferences -> Fetch remotes automatically
File -> Preferences -> Tab indicator for uncommitted changes

**Comment by fref** [comment]
Must have been that tab indicator thing, thank you.

**Comment by DanPristupov** [comment]
Yes. In addition to this, I'd recommend to:
- Run `git gc` in terminal
- Remove obsolete untracked files (if possible)

**Comment by fref** [comment]
Actually, it still does some sort of background operations.
I do most of my git manipulations on the CLI, switching between many branches, I use Fork mostly to get a better visual overview of the intricate branch tree. When I do some changes (pull, fetch, switching branches, ...), it still does some background checks to refresh itself. I'd like to be able to have the refresh triggered manually.

**Comment by DanPristupov** [comment]
Do you mean, Fork refreshes the data after fetch? Yes, it does, otherwise the user won't see the changes.

**Comment by fref** [comment]
It seems to be watching the directory and does some refresh when a git
action happened on the cli which modified something.
Since I do a lot of swapping between branches (many PR to review, rebase,
...), I'd prefer the option to disable this and only have it refresh on F5,
for instance.

**Comment by fref** [comment]
It seems to be watching the directory and does some refresh when a git action happened on the cli which modified something.
Since I do a lot of swapping between branches (many PR to review, rebase, ...), I'd prefer the option to disable this and only have it refresh on F5, for instance.

**Comment by DanPristupov** [comment]
> It seems to be watching the directory and does some refresh when a git action happened on the cli which modified something.

No, Fork doesn't watch FS. It only updates when the window gets focus.

**Comment by fref** [comment]
Could this be made optional, then?
